requirements.txt

```diff
@@ -12,7 +12,6 @@ ddtrace
 sqlparse==0.5.0
 scalar-fastapi==1.0.3
 httpx>=0.27.0,<1
-aiopath>=0.7.7,<1
```

Comment:
dangling dependency I guess ?

Reply:
I've added it and went back from using it